doc/nrf/app_dev/device_guides/nrf70/wifi_advanced_security_modes.rst

```diff
@@ -65,6 +65,195 @@ You can use any AAA server for testing purposes, such as FreeRADIUS or hostapd.
+
+#. Hostapd installation
+
+    To install hostapd, use the following commands:
+
+.. code-block:: console
+
+     git clone git://w1.fi/hostap.git
+
+     cd hostap/hostapd
+
+     cp defconfig .config
+
+#. Edit the :file:`.config` file for hostapd to use it as a RADIUS server by using the following commands:
+
+   .. code-block:: console
+
+      Comment (by adding #) the following configurations
+      #CONFIG_DRIVER_HOSTAP=y
+      #CONFIG_DRIVER_NL80211=y
+      #CONFIG_LIBNL32=y
+
+      Enable the following configurations (by removing # from the front)
+      CONFIG_DRIVER_NONE=y
+      CONFIG_RADIUS_SERVER=y
+      CONFIG_EAP_PSK=y
+      CONFIG_EAP_PWD=y
+      CONFIG_EAP_GPSK_SHA256=y
+      CONFIG_EAP_FAST=y
+
+      Add the following configurations
+      CONFIG_PEERKEY=y
+      CONFIG_IEEE80211W=y
+
+      Verify required EAP Types are enabled
+      "CONFIG_EAP=y"
+      "CONFIG_EAP_TLS=y"
+      "CONFIG_EAP_PEAP=y"
+      "CONFIG_EAP_TTLS=y"
+
+#. Build the hostapd executable
+
+    .. code-block:: console
+
+        make clean ; make
+
+#. Copy the certificates for EAP-TLS to the hostapd folder by using the following commands:
+
+   .. code-block:: bash
+
+      cp zephyr/samples/net/wifi/test_certs/*  hostap/hostapd/
+
+      touch hostapd.eap_user_tls
+
+      vim hostapd.eap_user_tls
+
+      $ cat hostapd.eap_user_tls
+      # Phase 1 users
+      *       TLS
+
+      touch tls.conf
+
+      vim tls.conf
+
+      $ cat tls.conf
+      # Building hostapd as a standalone RADIUS server
+      driver=none
+      # RADIUS clients configuration
+      radius_server_clients=hostapd.radius_clients
+      radius_server_auth_port=1812
+      # Enable eap_server when we use hostapd integrated EAP server instead of external RADIUS authentication
+      eap_server=1
+      # EAP server user database
+      eap_user_file=hostapd.eap_user_tls
+      # CA certificate
+      ca_cert=ca.pem
+      # Server certificate
+      server_cert=server.pem
+      # Private key matching with the server certificate
+      private_key=server-key.pem
+      # Passphrase for private key
+      private_key_passwd=whatever
+      logger_syslog=-1
+      logger_syslog_level=2
+      logger_stdout=-1
+      logger_stdout_level=2
+      ctrl_interface=/var/run/hostapd
+      ctrl_interface_group=0
+
+      vim hostapd.radius_clients
+
+      $ cat hostapd.radius_clients
+      RADIUS client configuration for the RADIUS server
+      0.0.0.0/0       whatever
+
+#. Run the hostapd
+
+    Run hostapd by using the following commands, assuming that **eno1** is the laptop interface connected to the AP (Authenticator) through Ethernet.
+
+.. code-block:: bash
+
+    ./hostapd -i eno1 tls.conf
+
+    or
+
+    To enable debug messages and Key data
+    ./hostapd -i eno1 tls.conf -ddK
+
```

Comment:
Do we need to add # before "To enable debug messages and Key data" to comment out?
```suggestion
Run hostapd by using the following commands, assuming that **eno1** is the laptop interface connected to the AP (authenticator) through Ethernet.

 .. code-block:: bash

    ./hostapd -i eno1 tls.conf

    or

    To enable debug messages and Key data
    ./hostapd -i eno1 tls.conf -ddK
```